journal/original/12-ancient-path.adoc

```diff
@@ -63,10 +64,13 @@ If that graciously attend,
 though what is here offered may be justly accounted as the barley-loaves,
 the hungry soul may receive some strength and refreshment thereby.
 
+[.signed-section-context-close]
 30th of Eighth month, 1764.
 
+[.centered]
```

Comment:
I think this will look a little better if we use the [.old-style] subheading, so that the subtitle is centered and not just indented. That way, when you get to the shorter subtitles in section 4 and 5, they will all still be centered. You can find it in the style guide, but the basic idea would be 

[.old-style]
=== Section I / Containing Tender Advice, Caution and Counsel to Parents and Children